docs/usage/help.rst.inc

```diff
@@ -156,7 +156,7 @@ Examples::
 
         Via ``--pattern`` or ``--patterns-from`` you can define BOTH inclusion and exclusion
         of files using pattern prefixes ``+`` and ``-``. With ``--exclude`` and
-        ``--exlude-from`` ONLY excludes are defined.
+        ``--exclude-from`` ONLY excludes are defined.
```

Comment:
this change should be removed in **this** file - it will be regenerated from source at release time.

Reply:
i do not see this being fixed in the source, likely in archiver.py.

so, please check if you fixed all the stuff you found in the source (not generated) file.

Reply:
ehrm, now i see it. maybe looked at wrong diff?

Reply:
still, it needs to be removed here.

Reply:
i reverted this change in the autogenerated file.